tests/testcachedbitmap.c

```diff
@@ -0,0 +1,131 @@
+#ifdef WIN32
+#ifndef __cplusplus
+#error Please compile with a C++ compiler.
+#endif
+#endif
+
+#if defined(USE_WINDOWS_GDIPLUS)
+#include <Windows.h>
+#include <GdiPlus.h>
+
+#pragma comment(lib, "gdiplus.lib")
+#else
+#include <GdiPlusFlat.h>
+#endif
+
+#if defined(USE_WINDOWS_GDIPLUS)
+using namespace Gdiplus;
+using namespace DllExports;
+#endif
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "testhelpers.h"
+
+#define C(func) assert (func == Ok)
```

Comment:
This definition doesn't match the usage later (1 parameter here, 2 parameters later). It would be nice to stick to the same test macros as in the other test files.

Reply:
Must have made a mistake when fixing the formatting. As for the macro itself, I copied it straight out of https://github.com/mono/libgdiplus/blob/9efff6a9ed1c3286362270b704dcdcf7a2670d54/tests/testclip.c#L16